### PR TITLE
[3.0.3] Add additional prpoperties

### PR DESCRIPTION
**Pull request by ashishgupta6**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,35 +118,57 @@ override fun onCreate() {
...
+attributes.put("PREFERRED_LANGUAGE", "English/Spanish/etc.");
+
+UpdateOptions updateOptions = new UpdateOptions.Builder()
+       .setUserId("<user-id>") // Set user id here, you can reference the result in future
+       .setPhoneNumber("<1234567890>") // you can add the customer phone number
+       .setPhoneInputType(PhoneInputType.PASTED) //See PhoneInputType class
+       .setOtpInputType(OtpInputType.AUTO_FILLED) //See OtpInputType class
+       .setUserEventType(UserEventType.LOGIN) //See UserEventType class
+       .setMerchantId("<merchant_id>") // Set Merchant Id
+       .setAdditionalAttributes(additionalAttributesTransaction) // Set attributes to store key-value pairs.
```
Comment (anz3r) [inline README.md:143]: rename

File: README.md [hunk 2]
```diff
@@ -118,35 +118,57 @@ override fun onCreate() {
...
 
 val updateOptions = UpdateOptions.Builder()
         .setUserId("<user-id>") // Set user id here, you can reference the result in future
         .setPhoneNumber("<1234567890>") // you can add the customer phone number
         .setPhoneInputType(PhoneInputType.PASTED) // See PhoneInputType class
         .setOtpInputType(OtpInputType.AUTO_FILLED) // See OtpInputType class
         .setUserEventType(UserEventType.LOGIN) // See UserEventType class
-        .setMerchantId("<merchant_id>") // Set Merchant Id 
+        .setMerchantId("<merchant_id>") // Set Merchant Id
+        .setAdditionalAttributes(additionalAttributesTransaction) // Set attributes to store key-value pairs.
```
Comment (anz3r) [inline README.md:171]: rename

File: README.md [hunk 3]
```diff
@@ -118,35 +118,57 @@ override fun onCreate() {
 <br>
 
 ## Optional Parameters
-You can add optional parameters like userId anytime and update the instance of Sign3Intelligence.
+1.	You can add optional parameters like UserId, Phone Number, etc., at any time and update the instance of Sign3Intelligence.
+2. To update the Sign3Intelligence instance with optional parameters, including additional attributes, you can use the following examples.
+3. **NOTE:** You need to call **getIntelligence()** function whenever you update options.
```
Comment (anz3r) [inline README.md:123]: callout that once options are updated, it gets reset. Clients need to explicitly update the options again to ingest again or else default value of OTHERS will be sent to backend

File: README.md [hunk 4]
```diff
@@ -118,35 +118,58 @@ override fun onCreate() {
 <br>
 
 ## Optional Parameters
-You can add optional parameters like userId anytime and update the instance of Sign3Intelligence.
+1.	You can add optional parameters like UserId, Phone Number, etc., at any time and update the instance of Sign3Intelligence.
+2. To update the Sign3Intelligence instance with optional parameters, including additional attributes, you can use the following examples.
+3. Once the options are updated, they get reset. Clients need to explicitly update the options again to ingest them, or else the default value of OTHERS will be sent to the backend.
```
Comment (anz3r) [inline README.md:123]: default value of OTHERS in userEventType

Comment (anz3r) [reply README.md:123]: move point 2 to last as examples make sense just after that point

Comment (ashishgupta6) [reply README.md]: Updated all the points.

**Review by anz3r — APPROVED** [review]

**Pull request merged**